### PR TITLE
Practice for chapter5

### PR DESCRIPTION
**Pull request by VuThiTranVan**
Filling in the layout
Get an introduction to numbers, Strings, properties, and methods

### CODE REVIEW

**Comment by VuThiTranVan** [comment]
screenshot from 2017-10-06 12-54-47

**Comment by VuThiTranVan** [comment]
ready

File: sample_app/Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ git_source(:github) do |repo_name|
 end
 
 gem "rails", "~> 5.1.4"
+gem 'bootstrap-sass', '3.3.7'
```
Comment (dothidiemthao) [inline sample_app/Gemfile]: Nháy "

**Comment by dothidiemthao** [comment]
commented

**Comment by VuThiTranVan** [comment]
screenshot from 2017-10-06 13-15-34

**Comment by VuThiTranVan** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Pull request merged**